index.html

```diff
@@ -3,14 +3,88 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
 </head>
 
-<body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
-</body>
+<div class="container">
+
+    <header>
+        <nav>
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </nav> 
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> 
+      </header>
+
+  <body>
+
+    <section class="hero">
+      <div class="hero-text">
+        <h1> <br>Innovation<br> On<br>Demand</h1>
```

Comment:
```suggestion
        <h1> Innovation<br> On<br>Demand</h1>
```

The br at the beginning of the h1 was what was causing that huge extra space